px/lg/templates/deployment.yaml

```diff
@@ -33,6 +33,13 @@ spec:
       - name: {{ $.Values.global.region }}-pxrs-{{ $lg }}
         image: keppel.{{ $.Values.registry }}.cloud.sap/{{ $.Values.lg_image }}
         imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: {{ $.Values.resources.lg.requests.cpu }}
+            memory: {{ $.Values.resources.lg.requests.memory }}
+          limits:
+            cpu: {{ $.Values.resources.lg.limits.cpu }}
+            memory: {{ $.Values.resources.lg.limits.memory }}
```

Comment:
If the structure of the values file matches the structure of the manifest, you can just include the values like here:
https://github.com/sapcc/helm-charts/blob/master/px/bird/templates/_deployment-bird.tpl#L63

No need to translate each individual attribute.